### ISSUE TITLE
The Registry's .txt Raw Format Incorrectly Renders Location, Phone, and Email Information

### ISSUE THREAD

**Issue by tylermneher** [jsonresume/jsonresume.org#81]
The registry had a _Raw Format_ functionality mentioned [here](https://jsonresume.org/getting-started/) that can render one's gist named `resume.json` as json, yaml, and txt formats. The json and yaml renderings are fine, but renderings in the txt format incorrectly renders the user's location, phone, and email keys from their gist. 

The location key renders with the placeholder:
`Pittsburgh, PA 15201`, the phone key renders with the placeholder `(555) 555-5555`, and the email key renders with the placeholder`example@example.com`.

Examples:

1. Navigating to the url `https://registry.jsonresume.org/tylermneher.txt` to render my (@tylermneher) resume in txt format returns the following:

```

Tyler M. Neher
Pittsburgh, PA 15201
(555) 555-5555
example@example.com

PROFESSIONAL SUMMARY
============================
Cybersecurity professional with a background in data science and software development.


WORK HISTORY
============================

2021-01-14 – undefined
Neher Data Systems
Owner

Providers of bespoke and customized technology solutions for B2C, B2B, and B2G clients and customers. Multi-jurisdiction and SAM Federally verified contractor of data science, cybersecurity, database engineering, and software development consulting services and solutions. Enterprise integrators and MSPs for multiple leading SaaS, PaaS, and IDaaS offerings.



,
2021-06-14 – 2022-11-01
AT&T, Inc.
Senior Cybersecurity

Responsible for cyber security areas across products, services, infrastructure, networks, and/or applications while providing protection for AT&T, its customers and its vendors/partners. Works with senior team members on various projects relating to the protection of devices, customers, assets, data, information technology, and networks. Supports innovation, strategic planning, technical proof of concepts, testing, lab work, and various other technical program management related tasks associated with the cyber security programs.



,
2019-09-01 – 2020-03-01
Memorial Sloan Kettering Cancer Center
Correlative Science Associate

Managed twenty-five unique, complex correlative registries, non-therapeutic trials, and correlative studies included in the Prostate Cancer Clinical Trials Consortium (PCCTC)  research portfolio. Developed and implemented improved data management systems and external reporting tools using Excel, VBA, Python 3, SQL, and PowerBI.



,
2017-06-01 – 2019-04-01
Hackensack Meridian Health
Biorepository Technologist

Managed the acquisition, processing, and handling of over three-thousand surgically resected solid tumor samples destined for storage in the HMH Biorepository. Implemented and administered the BioFortis Labmatrix research database system across our six campus sites. Was the study lead and federal project manager’s point person onsite for our NIH/NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) Phase III solid tumor collections contract.





VOLUNTEER
============================

2024-04-16 – undefined
Borough of Tinton Falls, New Jersey 
Green Team Advisory Committee Appointed Member

The Green Team works with the Governing Body, Environmental Commission and Sustainable Jersey, a nonprofit organization that provides tools, training and financial incentives to support communities as they pursue sustainability programs. These programs are designed to foster resiliency, reduce waste and improve the environment for future generations.

+ Appointed on the 16th of April 2024.
+ Appointment is for a one-year term.


,
2019-09-27 – undefined
Permanent Mission of Saint Kitts and Nevis to the United Nations  
General Assembly Delegation Member

Active member of the delegation from the Federation of Saint Kitts and Nevis to the United Nations General Assembly in New York.

+ 74th General Assembly meeting with the Federation's Prime Minister Dr. Timothy S. Harris.
+ 77th General Assembly meeting the Federation's Prime Minister Dr. Terrance M. Drew.



EDUCATION
============================

2010-09-01 – 2014-05-01
University of Central Florida 
Biology and Zoology - Bachelors of Science




,
2020-01-01 – 2020-07-01
Rutgers University 
Data Science - Certificate





AWARDS
============================

Microsoft Azure AI Fundamentals [AI-900] – 2024-05-04
Microsoft 

,
Microsoft Azure Fundamentals [AZ-900] – 2022-09-02
Microsoft 


CERTIFICATES
============================

Microsoft Azure AI Fundamentals [AI-900] – 2024-05-04
Microsoft 


,
Microsoft Azure Fundamentals [AZ-900] – 2022-09-02
Microsoft 



  
SKILLS
============================

Data Science Platforms and Tools
-------------
+ Python3
+ Anaconda
+ Julia
+ Pandas
+ Numpy
+ Pyenv
+ Conda
+ Jupyter
+ Plotly
+ Relational Database Systems
+ SQL
+ Postgres
+ noSQL
+ MongoDB
+ AWS RDS
+ SQLAlchemy
+ Databricks


Microsoft
-------------
+ Microsoft Azure
+ Entra
+ Exchange Online
+ MSAL
+ Microsoft Graph
+ Microsoft 365
+ OneDrive for Business
+ Teams
+ To Do
+ Bookings
+ Stream
+ Visio
+ PowerPoint
+ Planner
+ Power Automate
+ Power Apps
+ Project
+ Access
+ Outlook
+ Excel
+ Word
+ OneNote
+ SharePoint Online
+ Forms
+ Lists


Business Intelligence and Dashboarding
-------------
+ PowerBI
+ PowerBI Server
+ Tableau
+ Tableau Server
+ Plotly


Web Development Tools and Security
-------------
+ HTML
+ CSS
+ JavaScript
+ ReactJS
+ OIDC
+ SAML
+ SSO
```

2. Navigating to the url `https://registry.jsonresume.org/thomasdavis.txt` to render the resume for @thomasdavis in txt format returns the following:
```

Thomas Davis
Pittsburgh, PA 15201
(555) 555-5555
example@example.com

PROFESSIONAL SUMMARY
============================
I'm a full stack web developer who can build apps from the ground up. I've worked mostly at startups so I am used to wearing many hats. I am a very product focused developer who prioritizes user feedback first and foremost. I'm generally very flexible when investigating new roles. 


WORK HISTORY
============================

2020-05-05 – undefined
Tokenized
Senior Javascript Developer

Tokenized is a Bitcoin wallet for issuing, managing and trading digital tokens. I built out the front end which was packaged as an electron app. It was a difficult frontend to build because we store the users keys locally and used them to sign transactions and contracts.

+ React
+ Redux
+ SCSS
+ Product

,
2018-03-01 – 2020-01-01
Blockbid
Senior Javascript Developer

Blockbid is an Australian crypto currency exchange. I started off on the frontend but eventually became lead tech and worked on every moving piece of the exchange. I really enjoyed working with liquidity providers and connecting their platforms to help us achieve liquid markets.

+ React, Apollo, Styled Components
+ Node.js / Rails
+ Docker / Heroku / GCP
+ Used Figma for design and UX work
+ Optimizing markets with tens of millions of rows using SQL.

,
2016-01-01 – 2018-01-01
Listium
Developer

Built a very large and complex React / Redux application. It works on all platforms and has IOS/Android builds due to it being a PWA. (wrapped it in React Native though only implementing a WebView)

+ Worked with Postgres, Redis and Dynamodb for storage.
+ Hosted on a mixture of Heroku Apps and EC2 servers.
+ Caching by Fastly and Cloudflare
+ Hybrid app supported on all platforms

,
2011-01-08 – undefined
Cdnjs
Co-Founder

Following Google's CDN for jQuery, we decided to start a CDN for the less popular Javascript frameworks. The CDN is community moderated and open source on GitHub. We secured a partnership with Cloudflare who now supports the infrastructure.

+ Millions of sites use the CDN in production
+ Larger market share than Yahoo's, Microsoft's and Google's javascript content distribution networks
+ We serve hundreds of billions request a month
+ Contains over 3000 popular Javascript libraries
+ Millions of developers visit the site per year

,
2014-04-01 – 2016-01-01
Electronic Frontier Foundation
Developer

Hired to take EFF's campaigning to the next level by building an action centre in Ruby on Rails. The action centre is built around some large open source tools that lower the barrier to entry when contacting congress.

+ Developed new tools for contacting congress
+ Brainstormed campaign ideas to raise maximum awareness about issues
+ Lots of social networking integration

,
2013-03-08 – 2015-01-09
Earbits
CTO

Started off as a front end developer but took on the role of CTO in early 2013. The application frontend is built with Javascript and organized as a single page application that talks to a collection of Rails web servers which are connected to MongoDB.

+ Managed a small team of developers and designers
+ Built the entire frontend application with Backbone.js
+ Transferred all of the infrastructure from Heroku to AWS

,
2014-04-01 – undefined
JSON Resume
Founder

JSON Resume is a community driven open source initiative to create a JSON based standard for resumes. There is no reason why there can't be a common standard for writing a resume that can be extended with an ecosystem of open source tools.

+ This resume is built with JSON Resume
+ Over 3000 stars on Github
+ Community developed themes
+ Tens of thousands of users

,
2013-06-01 – 2016-01-01
Taskforce.is
Developer

Worked on many politically charged campaigns against mass surveillance. Not only technically challenging work but also a lot of networking and getting my hands dirty with politics.    Our biggest project was "TheDayWeFightBack"..

+ Generated  37,000,000 banner views
+ 100, 000 phone calls to congress
+ 500, 000 emails
+ 250, 000 signatures

,
2013-01-01 – undefined
Drone Hire
Co-Founder

An international directory of civilian drone / UAV operators for hire. Services include aerial photography, aerial video, mapping, surveying, precision agriculture, real estate photography, remote inspections and infrared imaging. Our plan is to be the place to go when looking for UAV/Drone services. The website is built in Backbone.js and API is built with Node.js and Postgres. 

+ The site and blog combined have managed to receive over 200,000 visitors in 2014.

,
2011-01-01 – 2014-01-01
BackboneTutorials.com
Founder

I write tutorials and blog post regarding the popular Javascript framework Backbone.js. The tutorials cover a range of topics regarding front end development aimed at beginners, experts and anyone in between.

+ Over two million unique visitors a year
+ 25,000+ ebook downloads
+ 300,000+ Youtube views

,
2011-01-01 – 2012-06-01
Ephox
Front-end Developer

Ephox is a worldwide company who is heavily involved with the development of TinyMce and enterprise editors. My primary role included building front-end widgets and applications. Worked on a major product using Backbone.js as a base. Heavily involved in UI/UX design and wire-framing. Also spend a lot of time designing API specifications and documentation.




EDUCATION
============================

2008-02-12 – 2009-12-12
The University of Queensland 
Software Engineering (incomplete) - Bachelors





AWARDS
============================

Defender of the Internet – 
Fight For The Future 



SKILLS
============================

Frontend
-------------
+ HTML / JSX
+ SCSS / CSS / BEM / Styled Components
+ Javascript / Typescript
+ React / Next
+ Redux / Apollo


Backend
-------------
+ Node
+ Ruby
+ Python
+ Postgres
+ Redis
+ Serverless


Devops
-------------
+ AWS
+ G Cloud
+ Heroku
+ Caching
```

**Comment by tylermneher** [comment]
@thinkula heads up on this bug

**Comment by tylermneher** [comment]
I opened a pull request [Resume Registry Raw Format .txt rendering fix Ref: Issue #81 #82](https://github.com/jsonresume/jsonresume.org/pull/82) that should fix this bug. Just waiting on review!

**Comment by thomasdavis** [comment]
Thanks again @tylermneher that has been merged
